app/assets/javascripts/oxalis/controller/combinations/volumetracing_plane_controller.js

```diff
@@ -139,6 +140,25 @@ class VolumeTracingPlaneController extends PlaneControllerClass {
 
         this.volumeTracingController.handleCellSelection(cellId);
       },
+
+      mouseMove: (delta: Point2, position: Point2) => {
+        const tool = getVolumeTool(Store.getState().tracing).get();
+        if (tool === VolumeToolEnum.BRUSH) {
+          Store.dispatch(setBrushPositionAction([position.x, position.y]));
+        }
+      },
+
+      out: () => {
+        Store.dispatch(hideBrushAction());
+      },
+
+      scroll: (delta: number, type: ?ModifierKeys) => {
+        if (type === "shift") {
```

Comment:
Shouldn't this also check whether the brush tool is active? Otherwise, scrolling in another mode will change the brush size?